sub try()
	on error goto onException
end sub
